scripts/run_tests.sh

```diff
@@ -0,0 +1 @@
+time python -m unittest
```

Comment:
You don't need `time` here. 

Side point, but I think `time` does something different on windows terminal than linux and so if we want to avoid some potential incompatibilities, better to leave that out.